.travis.yml

```diff
@@ -23,13 +23,13 @@ matrix:
           sudo: required
           dist: trusty
           group: edge
-        - php: 5.3
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
+      #  - php: 5.3
+       # - php: 5.4
+        #- php: 5.5
+        #- php: 5.6
         - php: 7.0
-          env: deps=high
-        - php: 7.1
```

Comment:
We need to keep a PHP 7.1 job (even though it is not one doing a special handling of deps). Otherwise, Symfony would be untested on 7.1, which is not acceptable.